Workload-Security/Deployment/docker-ws-binary-generator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:latest
+
+RUN apt update -y && apt install -y tar wget gcc make
+
+WORKDIR /root
+
+RUN wget https://github.com/neurobin/shc/archive/release.tar.gz && tar -xvf release.tar.gz && cd shc-release && ./configure && make && make install && cd .. && rm -rf shc-release release.tar.gz
+
+ADD ws.sh /root/ws.sh
```

Comment:
I would be better but the script that im using here is not the original, I had to make some modifications because the tenantid and token are hardcoded, so I made them variables so we can use and pass local environment variables in the container at runtime.